src/components/AttentionBox/__tests__/attentionBox-tests.jest.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import AttentionBox from "../AttentionBox";
+
+describe("AttentionBox tests", () => {
+  const title = "title";
+  const text = "text";
+  let onCloseMock;
+  let attentionBoxComponent;
+  
+  beforeEach(() => {
+    onCloseMock = jest.fn();
```

Comment:
please move the creation of onCloseMock and the setting of this function as the onClose prop as part of the onClose test and not part of the beforeEach function